script.js

```diff
@@ -0,0 +1,17 @@
+const menu = document.querySelector('.menu');
+const menuList = document.querySelector('.t-menu');
+const close = document.querySelector('.c-button');
+const lists = document.querySelector('.desktop-nav-bars');
+
+menu.addEventListener('click', () => {
+  menu.classList.toggle('active');
+  menuList.classList.toggle('active');
+});
+
+close.addEventListener('click', () => {
+  menuList.classList.remove('active');
+});
+
+lists.addEventListener('click', () => {
+  menuList.classList.remove('active');
+});
```

Comment:
The mobile nav menu should close automatically when I click a link for the menu. Please fix this issue.